File: Android Java Retrofit Node Express- Orphanslife/app/src/main/java/com/kaligotla/oms/AdminView/Location/Location.java
package com.kaligotla.oms.AdminView.Location;

import java.io.Serializable;

public class Location implements Serializable {
    private int id, pincode;
    private String area, city, district, state;

    public Location() {
    }

    public Location(int pincode, String area, String city, String district, String state) {
        this.pincode = pincode;
        this.area = area;
        this.city = city;
        this.district = district;
        this.state = state;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getPincode() {
        return pincode;
    }

    public void setPincode(int pincode) {
        this.pincode = pincode;
    }

    public String getArea() {
        return area;
    }

    public void setArea(String area) {
        this.area = area;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getDistrict() {
        return district;
    }

    public void setDistrict(String district) {
        this.district = district;
    }

    public String getState() {
        return state;
    }

    public void setState(String state) {
        this.state = state;
    }

    @Override
    public String toString() {
        return ""+pincode+"\n"+area+"\n"+city+"\n"+district+"\n"+state;
    }
}

File: Android Kotlin Retrofit Node Express- Orphanslife/app/src/main/java/com/kaligotla/orphanslife/data/model/Location.kt
package com.kaligotla.orphanslife.data.model

import java.io.Serializable


class Location : Serializable {
    var id = 0
    var pincode = 0
    var area: String? = null
    var city: String? = null
    var district: String? = null
    var state: String? = null

    constructor() {}
    constructor(id: Int) {
        this.id = id
    }

    constructor(pincode: Int, area: String?, city: String?, district: String?, state: String?) {
        this.pincode = pincode
        this.area = area
        this.city = city
        this.district = district
        this.state = state
    }
}

